sys/net/application_layer/dhcpv6/client.c

```diff
@@ -828,6 +1055,16 @@ static void _request_renew_rebind(uint8_t type)
                     mrd = valid_until;
                 }
             }
+            if (IS_USED(MODULE_GNRC_DHCPV6_CLIENT_IA_NA)) {
+                /* calculate MRD from addr_leases */
+                for (unsigned i = 0; i < CONFIG_DHCPV6_CLIENT_ADDR_LEASE_MAX; i++) {
```

Comment:
Same as in https://github.com/RIOT-OS/RIOT/pull/16658 to keep indentation at bay:

```suggestion
            /* calculate MRD from addr_leases */
            for (unsigned i = 0;
                 IS_USED(MODULE_GNRC_DHCPV6_CLIENT_IA_NA) && 
                 (i < CONFIG_DHCPV6_CLIENT_ADDR_LEASE_MAX);
                 i++) {
```

Reply:
I incorporated this suggestion via 14433a1bd045080bb164c864ce241a68624f3e40